pages/index.js

```diff
@@ -26,7 +26,7 @@ class Index extends React.Component {
       updateQueryString(this.props.router, state)
       saveSettings(
         localStorage,
-        omit(state, ['code', 'backgroundImage', 'backgroundImageSelection', 'filename'])
+        omit(state, ['code', 'backgroundImage', 'backgroundImageSelection', 'filename', 'gist'])
```

Comment:
Do you think it is worth having a separate component state (somewhere) that holds all the gist state? That way the editor doesn't have to omit all of that?